### ISSUE TITLE
"You have been blocked" v2.3.0

### ISSUE THREAD

**Issue by pythonInRelay** [Mastermindzh/tidal-hifi#68]
When using the latest .deb file

> You have been blocked.
> Why this blocking? Something about the behaviour of the browser has caught our attention.
> 
> There are various possible explanations for this:
>  you are browsing and clicking at a speed much faster than expected of a human being
>  something is preventing Javascript from working on your computer
>  there is a robot on the same network (IP {redacted}) as you
> Having problems accessing the site? Contact support

**Comment by Sivecano** [comment]
Same issue here, Tidal works in my browser but not in the app. I can look at the start page but not log in. as soon as I try, I get the message, that I've been blocked

**Comment by pythonInRelay** [comment]
Yeah it has nothing to do with IP. Using a normal IP, no VPN or anything else. Blocks me no matter what package I use. It's due to the implementation of the browser.

**Comment by Sivecano** [comment]
ok, today it randomly started working. I genuinely don't know what's causing this error because I'm in the same place using the same WiFi as last time. I'm also using the non-git AUR package which hasn't been updated in the mean time either. perhaps it's related to me having had tidal open in a browser tab? perhaps it got confused by me making different requests from the same computer/account but with different browsers?

**Comment by Sivecano** [comment]
perhaps tidal changed their website?

**Comment by Mastermindzh** [comment]
The people blocking you are from Tidal (probs automated tho), how or why is unbeknownst to me.

Maybe the next time someone is blocked they can ask Tidal what the suspicious behavior would be and maybe that'll give us a fix.

And yeah, the tidal-hifi package that isn't the -git version is no longer maintained by myself. 
The person that took it has actually taken over other packages too and i.m.o cares more about the status of owning a few packages than actually having packages be up-to-date.

Best to use the -git version, it'll stay up to date :)

**Comment by pythonInRelay** [comment]
I just built the source and it works fine. I am not blocked at all.

I then installed the .deb and I am not blocked here either.

I am guessing this is an issue with the package version that is not a direct copy/link to this git version. So I think wherever people download that package, if it is not sourced from here. It should be reported as not working.

I am going to close this ticket as the dev is correct. This is not a problem with his source.
